package.json

```
@@ -20,14 +20,39 @@
   "author": "shamofu",
   "license": "MIT",
   "build": {
+    "productName": "shamofu",
+    "appId": "org.shamofu.electronReactParcelBoilerplate",
```

```suggestion
    "appId": "org.shamofu.ElectronReactParcel",
```
This is just my personal preference.